lk2nd/smp/cortex-a-msm8994.c

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I think to be sure this should be

```suggestion
// SPDX-License-Identifier: GPL-2.0-only
```

and moved to the gpl folder like krait is. Since this is pretty much a copy paste from downstream and we have no alternative documentation to take the sequence and register bits from.

Reply:
But now I'm not sure if it correct to add only me to the license header